_docs/principles/accessibility.md

```diff
@@ -0,0 +1,216 @@
+- 0.6% are blind (1 in 188 people)
+ 
+By using the following best practices, you can help ensure your design is accessible for the visually impaired.
+
+### Do not rely on color alone to convey meaning
+
+Color deficiency is a common issue faced in the design process. The most common form of color blindness is known as red and green color deficiency and most color deficient people suffer from either Protanopia (less sensitivity to red light) or Deuteranopia (less sensitivity to green light). Red and green color deficiency can mean that those affected have troubles distinguishing certain hues of red and green. For example, an individual with red or green color deficiency can confuse blue and purple as they may not be able to distinguish the red element of purple.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+{% figure [caption:"Helix colors as seen in normal vision _(left)_; Protanopia - reduced sensitivity to red _(center)_; Deuteranopia - reduced sensitivity to green _(right)_."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/overview/accessibility/accessibility-hero.svg)
+{% endfigure %}
+
+While color can be useful to convey information, color should not be the only way information is conveyed. When using color to differentiate elements, also provide additional identification that does not rely on color perception. There should always be a backup visual indicator for people with color blindness. 
+
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Alerts
+
+Alert bars are a great example of how multiple indicators can be used. If you are indicating an error alert (for example, making it red) always include a status icon and alert description so that someone who is color blind can still see the problem.
+
+Other optional indicators to consider are:
+
+- **Alert type:** An optional alert title descriptor.
+- **Alert action link:** Provides a user with the next step of a process.
+- **Close action:** Closes the alert.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/overview/accessibility/accessibility-alerts.svg)
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Color and contrast
+
+Readability requires sufficient contrast between the text and its background. Low contrast is especially problematic for users with low vision who also often have contrast sensitivity. To ensure text is readable, Helix adheres to accessibility standards defined by [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG21/). These standards are based on contrast ratios calculated from text size, text color, and background color.  
```

Comment:
Start a new paragraph at "To ensure text is readable..."

Reply:
updated.